main.py

```diff
@@ -64,7 +64,11 @@ def trace(self, request: Request) -> JSONResponse:
             stored_flows = get_stored_flows()
         except tenacity.RetryError as exc:
             raise HTTPException(424, "It couldn't get stored_flows") from exc
-        result = self.tracepath(entries, stored_flows)
+        try:
+            result = self.tracepath(entries, stored_flows)
+        except ValueError as exc:
+            log.debug("tracepath error {exc}")
```

Comment:
@gretelliz if you could remove this log.debug here with the exc that'd be great. Our NApps don't typically log a traceback at the debug level if they're being handled as you've done correctly. Same comment to `log.debug` below.

Reply:
Done!